### PR TITLE
Add ECS-cluster module

### PR DESCRIPTION
**Pull request by MattiasGees**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
+ * [`cluster_name`]: String: The name of the ECS cluster.
+ * [`ecs-instance-profile`]: IAM instance profile that you need to give to your instances.
+ * [`ecs-service-role`]: IAM service role for you ECS to manage your loadbalancers
+
+### Example
+```
+module "ecs_cluster" {
+  source      = "ecs_cluster"
+  project     = "myproject"
+  environment = "production"
```
Comment (simonrondelez) [inline README.md]: an extra tag maybe?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,22 @@
 # terraform-ecs
 Terraform ECS module
+
+## ecs-cluster
+Setup a basic ECS cluster with the needed IAM rights.
+
+### Available variables:
+ * [`project`]: String(required): Project name
+ * [`environment`]: String(optional):  For what environment is this cluster needed (eg staging, production, ...)
```
Comment (simonrondelez) [inline README.md]: looking at the variables file its required

File: ecs-cluster/iam.tf [hunk 3]
```diff
@@ -0,0 +1,61 @@
+
```
Comment (simonrondelez) [inline ecs-cluster/iam.tf:1]: isn't this a bit to static?

**Review by SamClinckspoor — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ringods) [reply README.md]: Change the example to show it with the new environments:
`environment = "${terraform.env}"`

File: README.md [hunk 4]
```diff
@@ -1,2 +1,22 @@
...
+### Output
+ * [`cluster_name`]: String: The name of the ECS cluster.
+ * [`ecs-instance-profile`]: IAM instance profile that you need to give to your instances.
+ * [`ecs-service-role`]: IAM service role for you ECS to manage your loadbalancers
+
+### Example
+```
+module "ecs_cluster" {
+  source      = "ecs_cluster"
```
Comment (ringods) [inline README.md:18]: Could you change the source to include a full github URL? Easier to copy-paste the example. 😄

File: ecs-cluster/iam.tf (hunk 3 above)
Comment (ringods) [reply ecs-cluster/iam.tf:1]: @simonrondelez don't forget that you can link specific roles to your ECS containers. Not all roles & profiles need to end up on the instance.

**Review by ringods — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (MattiasGees) [reply README.md:18]: No because that will change, I rather leave it completely to not make any mistakes.

**Review by ringods — APPROVED** [review]

**Pull request merged**